.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+## Description
+<!--- Describe your changes in detail. -->
+
+## Related issue
+<!--- Add a link to the issue (follow the b/XXX format for internal issues) -->
+
+## How has this been tested?
+<!--- Please describe how you tested the changes besides the automatically triggered unit tests when applicable. -->
+<!--- Include sample output logs or metrics and/or screenshots of key results when applicable. -->
```

Comment:
Will it be too strong to say "Must Include"?

Reply:
Done